scripts/start-console.sh

```diff
@@ -6,19 +6,10 @@ PLUGIN_NAME="forklift-console-plugin"
 CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
 CONSOLE_PORT=${CONSOLE_PORT:=9000}
 INVENTORY_SERVER_HOST=${INVENTORY_SERVER_HOST:="http://localhost:8080"}
-MUST_GATHER_API_SERVER_HOST=${MUST_GATHER_API_SERVER_HOST:="http://localhost:9200"}
+MUST_GATHER_API_SERVER_HOST=${MUST_GATHER_API_SERVER_HOST:="http://localhost:8090"}
```

Comment:
the default ports are different then configured for kind cluster (`30080` and `30022`)

Reply:
30080 - the web console
30022 - the development ssh server used with the `forklift-console-plugin-dev` image

in the `yarn start:console[:flavor]` flow you suppose to run the inventory on your local machine, the kind deployment does not support it.

the `forklift-console-plugin-dev` does support using forklift running in the cluster, and there it uses cluster internal urls:
http://forklift-inventory.konveyor-forklift.svc.cluster.local:8080